policies/templates/gcp_gke_cluster_location_v1.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-cluster-location-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GKEClusterLocationConstraintV1
+        plural: gkeclusterlocationconstraintsv1
+      validation:
+        openAPIV3Schema:
+          properties:
+            location:
```

Comment:
nit: it's probably more appropriate to make it plural: "locations" since it's a list of locations?

Reply:
I guess allowed_locations would be more explicit